.github/workflows/ci.yml

```diff
@@ -92,7 +92,7 @@ jobs:
       fail-fast: true
       matrix:
         python-version: ["3.11"]
-        nautobot-version: ["2.0.3"]
+        nautobot-version: ["2.2.3"]
```

Comment:
should we update the plugin requirements to be specific that this is the minimum Nautobot version?